easybuild/easyconfigs/t/TopHat/TopHat-2.1.0-gompi-2021b-Python-2.7.18.eb

```diff
@@ -0,0 +1,44 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+# Modified by: Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'ConfigureMake'
+
+name = 'TopHat'
+version = '2.1.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://ccb.jhu.edu/software/%(namelower)s/'
+description = "TopHat is a fast splice junction mapper for RNA-Seq reads."
+
+toolchain = {'name': 'gompi', 'version': '2021b'}
+toolchainopts = {'cstd': 'gnu++98', 'pic': True}
+
+source_urls = ['http://ccb.jhu.edu/software/tophat/downloads/']
+sources = ['%(namelower)s-%(version)s.tar.gz']
+checksums = ['e7fa2b170408e785e4fafd0bccf4ecfab5151c480a2438dffa42183cfda71f99']
+
+builddependencies = [('Autoconf', '2.71')]
+
+dependencies = [
+    ('Python', '2.7.18'),
+    ('SAMtools', '1.14'),
+    ('Boost', '1.75.0'),
```

Comment:
@ItIsI-Orient The Boost 1.75.0 requirement is causing some "fallout" higher up in the dependency tree, so I wonder if patching TopHat to be compatible with Boost 1.77.0 is a viable approach. Did you look into that?

Reply:
Could try with `'static': True` in `toolchainopts`, and then check the `tophat` binary with `ldd` to see if Boost gets linked statically...

Reply:
hmm, the `tophat` executable is actually not being dynamically linked regardless of the toolchainopt, only `segment_juncs` dynamically links against `Boost`

Reply:
adding `'static': True` to `toolchainopts` does result in a static `segment_juncs`